.github/workflows/ci.yml

```diff
@@ -61,7 +61,7 @@ jobs:
           fi
       # Tree-shaking check
       - name: Treeshaking check
-        run: rush build --only @internal/check-treeshaking
+        run: rush build --only @internal-tools/check-treeshaking
```

Comment:
Can this stay as `@internal/check-treeshaking`? The reason for the `@internal` was from a security concern to avoid any scenario where someone creates a malicious package with that name and publishes it on npm - then somewhere down the line this is included as a dev dependency or someone does an npm install on the repo and it pull the malicious package code instead

Reply:
Done.

@internal doesn't seem to be handled specially by NPM? It's just an unlikely NPM prefix, right? I thought @internal-tools would also meet that criterion, and would let us keep tools and release-targeted packages separate.

But I agree that having multiple such prefixes will add confusion, so reverted to @internal.

Reply:
I remember reading somewhere about it being a reserved organization name... but I can't find any source on that...

If I try to make it at least it fails:
![image](https://user-images.githubusercontent.com/2684369/131383634-3c95ece2-71d0-4b6c-b6a1-7fa45201817c.png)